Description: Prevent internal libmpeg2 from being built at all times
 While fs-uae 2.6.2 upstream doesn't use its internal libmpeg2 library
 anymore, the Makefile still includes the 'libmpeg2' folder in 'SUBDIRS'
 which means that the internal library is still being built even though
 fs-uae is linked against the distriution's version of libmpeg2. This
 results in FTBFS on several architectures where the built-in libmpeg2
 library does not built cleanly. This patch removes 'libmpeg2' from
 'SUBDIRS' and prevents it from being built at any times.
 .

--- fs-uae-2.6.2+dfsg.orig/Makefile.am
+++ fs-uae-2.6.2+dfsg/Makefile.am
@@ -1,6 +1,6 @@
 # Makefile.am for FS-UAE
 
-SUBDIRS = libmpeg2
+# SUBDIRS = libmpeg2
 
 WARNINGS =
 
